modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/editor/BalloonEditor.js

```diff
@@ -42,6 +42,15 @@ const BalloonEditor = ({config = {}, contents, name, ...otherProps}) => {
 		title: false,
 	};
 
+	const imageDirectionShim = editorConfig.toolbarImage.includes(
+		'ImageAlign'
+	) && {
+		style: {
+			float: 'left',
```

Comment:
It works fine with `float: 'right'` too. I think We just need to put a defined directed float value to make it work.

Reply:
Setting the main editable container to `float` is pretty invasive, it might mess up layout on some pages. A less invasive CSS trick to fix `float` overlap is to add `overflow: auto` to container.

Reply:
Replacing the `float: 'left' | 'right'` styles with `overflow: auto` looks fine :)

Thanks!!!!

Reply:
See https://github.com/liferay-frontend/liferay-portal/pull/1339/commits/df033efedceb335c9f7712859befaa2bb5003145